frontend/static/messages/it.json

```
@@ -1250,6 +1252,11 @@
       "defaultIndicator": "Sorgente di input predefinita"
     },
     "main": {
+      "errors": {
```

Biome (2.4.4)

[error] The key errors was already declared.

(lint/suspicious/noDuplicateObjectKeys)